esmvalcore/preprocessor/_io.py

```diff
@@ -501,9 +500,10 @@ def _concatenate_overlapping_cubes(cubes):
                                     data_start_1.month, data_start_1.day,
                                     start_overlap.year, start_overlap.month,
                                     start_overlap.day)
+            # convert c1_delta scalar cube to vector cube, if needed
+            if c1_delta.data.shape == ():
+                c1_delta = iris.util.new_axis(c1_delta, scalar_coord="time")
             cubes = iris.cube.CubeList([c1_delta, cubes[1]])
-            logger.debug("Attempting concatenatenation of %s with %s",
```

Comment:
Did you want to remove this?

Reply:
nit at all, I was being silly - cheers for pointing it out, bud, will reinstate :+1: